File: Structural/Bridge/Bridge_Java/Runner.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Bridge_Java;

/**
 *
 * @author sashant
 */
public class Runner {
    public static void main(String[] args) {
        RemoteButton b1=new MuteRemote(new TVDevice(1,200));
        RemoteButton b2=new PauseRemote(new TVDevice(1,200));
        b1.buttonNinePressed();
        b2.buttonNinePressed();
        
    }
}

File: Structural/Bridge/Bridge_py/Runner.py
from GenericDevice import GenericDevice
from PauseRemote import PauseRemote
from MuteRemote import MuteRemote
from Remote import Remote
from TVDevice import TVDevice

remote1=MuteRemote(TVDevice(100,2))
remote2=PauseRemote(TVDevice(100,2))
remote1.buttonNinePressed()
remote2.buttonNinePressed()